components/02-molecules/registration/NameRegisteredAwaitingRecordsSettingComponent.tsx

```diff
@@ -43,12 +47,18 @@ export const NameRegisteredAwaitingRecordsSettingComponent = ({
       handleNextStep();
     }
 
+    let resolverAddress =
+      nameRegistrationData.ensResolver === EnsResolver.Custom &&
+      nameRegistrationData.customResolverAddress
+        ? nameRegistrationData.customResolverAddress
+        : ensResolverAddress[nameRegistrationData.ensResolver];
```

Comment:
Suggest this turns into a util function inside `useNameRegistration()` since it repeats multiple times across different components. e.g. `getResolverAddr()`

Reply:
done